components/fieldgroups/profile/profile-consentResultsv2.schema.json

```diff
@@ -0,0 +1,89 @@
+{
+  "meta:license": [
+    "Copyright 2023 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/mixins/profile-consentResultsV2",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Consent policies evaluation results map",
+  "type": "object",
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/profile"],
+  "description": "Consent policies evaluation result for individual",
+  "definitions": {
+    "consent-status": {
```

Comment:
Also, as discussed over the slack thread, two things to note here:
1. Although this is a valid JSON schema, let's make this inline to the object where this is referred because, the UI does not yet support adding custom properties under an object which is defined as a local definition which is the case here. This was a recent issue reported and we would have to wait for UI to fix this. 
2. FYI, Any definition need to have a "properties"  section to wrap the fields defined in it. This change won't be necessary if we take care of the point# 1 above.

Reply:
Can we fix this as we are planning to push to INT/STG tomorrow (01/05) ?
cc: @ankushkd , @vedhera